nucypher/blockchain/eth/signers/software.py

```diff
@@ -105,8 +104,7 @@ def sign_transaction(self, transaction_dict: dict) -> HexBytes:
 
 
 class ClefSigner(Signer):
-
-    DEFAULT_IPC_PATH = '~/Library/Signer/clef.ipc' if sys.platform == 'darwin' else '~/.clef/clef.ipc'  #TODO: #1808
+    DEFAULT_IPC_PATH = '~/Library/Signer/clef.ipc' if sys.platform == 'darwin' else '~/.clef/clef.ipc'  # TODO: #1808
```

Comment:
Any reason we didn't make this a Path as well? and L121?

Reply:
Ah, I simply missed that.